manifests/repository.pp

```diff
@@ -122,7 +122,7 @@
         $url = "${scheme}${address}"
       }
       if $os  == 'Ubuntu' {
-        unless $osrelease in ['12.04', '14.04', '16.04', '18.04', '20.04'] {
+        unless $osrelease in ['20.04'] {
```

Comment:
I think this is bad practice. IMO we should remove the whole check and people should rely on the metadata.json to identify if their OS is supported or not.

Reply:
that is a fair point, thought about it too. I dont know anything about bareos specifically, but is the location that gets built even really that specific and only works on LTS? But in the end it doesnt matter, I agree with you we should remove that whole check-fail construct

Reply:
there are a few more lets say odd choices in this module. I think I will clean those up in another PR